corehq/apps/data_analytics/malt_generator.py

```diff
@@ -63,25 +66,28 @@ def _get_malt_row_dicts(self, domain_name, monthspan, users_by_id):
             apps_submitted_for = get_app_submission_breakdown_es(domain_name, monthspan, users)
```

Comment:
Not directly related, but it looks like the chunked call above is redundant here, as `users_by_id` was already sliced into chunks of 1000 within `build_table`.

Reply:
Great point!

Reply:
See 8aa83be048ca7b3adc8b8d7ff04d7c250eddf10a